### PR TITLE
[Merged by Bors] - chore: fix names and skips in `AlgebraicGeometry.Morphisms.Basic`

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/Morphisms/Basic.lean [hunk 1]
```diff
@@ -236,11 +232,9 @@ theorem targetAffineLocallyOfOpenCover {P : AffineTargetMorphismProperty} (hP :
     exact (hP.1.arrow_mk_iso_iff (morphismRestrictOpensRange f _)).mpr h𝒰
 #align algebraic_geometry.target_affine_locally_of_open_cover AlgebraicGeometry.targetAffineLocallyOfOpenCover
 
--- set_option maxHeartbeats 1000000 in
-open List in
 theorem AffineTargetMorphismProperty.IsLocal.affine_openCover_TFAE
     {P : AffineTargetMorphismProperty} (hP : P.IsLocal) {X Y : Scheme.{u}} (f : X ⟶ Y) :
-    TFAE
+    List.TFAE
```
Comment (jcommelin) [inline Mathlib/AlgebraicGeometry/Morphisms/Basic.lean:237]: Why this change? I like it if the statement is just "The Following Are Equivalent"...
Do you need this for a technical reason?

File: Mathlib/AlgebraicGeometry/Morphisms/Basic.lean [hunk 2]
```diff
@@ -378,24 +370,22 @@ theorem AffineTargetMorphismProperty.IsLocal.targetAffineLocallyIsLocal
...
       rw [← affine_cancel_left_isIso hP.1 e] at h𝒰
       convert h𝒰 using 1
       simp
 #align algebraic_geometry.affine_target_morphism_property.is_local.target_affine_locally_is_local AlgebraicGeometry.AffineTargetMorphismProperty.IsLocal.targetAffineLocallyIsLocal
 
-open List in
 theorem PropertyIsLocalAtTarget.openCover_TFAE {P : MorphismProperty Scheme}
     (hP : PropertyIsLocalAtTarget P) {X Y : Scheme.{u}} (f : X ⟶ Y) :
-    TFAE
+    List.TFAE
```
Comment (jcommelin) [inline Mathlib/AlgebraicGeometry/Morphisms/Basic.lean:388]: Idem

File: Mathlib/AlgebraicGeometry/Morphisms/Basic.lean (hunk 1 above)
Comment (Parcly-Taxel) [reply Mathlib/AlgebraicGeometry/Morphisms/Basic.lean:237]: It saves on line count. `open List in` ends up being used for just this one `TFAE`.

Comment (jcommelin) [reply Mathlib/AlgebraicGeometry/Morphisms/Basic.lean:237]: Hmm, I don't find that a very compelling reason.

File: Mathlib/AlgebraicGeometry/Morphisms/Basic.lean (hunk 2 above)
Comment (Parcly-Taxel) [reply Mathlib/AlgebraicGeometry/Morphisms/Basic.lean]: Fine, I've reverted both.

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**